File: code-coaches/easy/argentina/argentina.py
from decimal import Decimal


def lower_currency_after_conversion(ars: Decimal, usd: Decimal) -> str:
    ars_to_usd_rate = Decimal("0.02")
    converted_usd = ars * ars_to_usd_rate

    if converted_usd < usd:
        return "Pesos"
    else:
        return "Dollars"


def main():
    # Get the input values from the user
    ars = Decimal(input())
    usd = Decimal(input())

    # Call the function to get the result
    result = lower_currency_after_conversion(ars, usd)

    # Print the result
    print(result)


if __name__ == "__main__":
    main()

File: code-coaches/easy/argentina/Argentina.java
import java.math.BigDecimal;
import java.util.Scanner;

public class Argentina {

    public static String lowerCurrencyAfterConversion(BigDecimal ars, BigDecimal usd) {
		BigDecimal arsToUsdRate = new BigDecimal("0.02");
	    BigDecimal convertedUsd = ars.multiply(arsToUsdRate);

        if (convertedUsd.compareTo(usd) == 1)
            return "Dollars";
        else
            return "Pesos";
    }

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);

		BigDecimal ars = input.nextBigDecimal();
		BigDecimal usd = input.nextBigDecimal();

        System.out.println(lowerCurrencyAfterConversion(ars, usd));

		input.close();
	}

}
